src/views/Admin.vue

```diff
@@ -11,7 +11,7 @@
     Current User: {{ currentUser && currentUser.email }} isAdmin: {{ isAdmin }}
     <br />
 
-    <div v-show="isAdmin">
+    <div v-if="isAdmin && users && userSettings">
```

Comment:
`v-if` here is better since it doesn't render ANY of the content, so doesn't trigger errors with unavailable data. Had to also check to see if the `users` and `userSettings` have gotten loaded.